### PR TITLE
Development workflow

### LINKED ISSUE #17: Convention and Development Cycle
Update `conventions.md` (some details are outdated):
- [ ] check details
- [ ] fix details

Include detailed description of the development cycle of helpwave:
- [ ] provide tutorial graphics
- [ ] show some screenshots with example issues

### PR DESCRIPTION
**Pull request by MaxSchaefer**
Closes #17

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
> The result of the spellchecking is now available under ...

Bildschirm­foto 2023-02-06 um 9 38 12 PM
Bildschirm­foto 2023-02-06 um 9 37 48 PM

File: development/CONVENTIONS.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+where `<issue-number>` is the number of the issue, `<issue-name>` is the name of the issue and `<pull-request-number>`
+is the number of the pull request.
+### Examples
+```text
+Merge from issue/1-add-the-ability-to-create-a-new-user into main (#1)
+```
+```text
+Merge from issue/2-fix-the-bug-with-the-user-creation into main (#2)
+```
+[…]
```
Comment (use-to) [inline development/CONVENTIONS.md:79]:
```suggestion
Merge commits should look like what github suggests in a pull request.
```

Comment (MaxSchaefer) [reply development/CONVENTIONS.md:79]: For example, the `services` repository prefers squashed merge commits. With that in mind, the commit message should be valid against our commit message convention.

I accept the suggestion for now, we didn't enforce this policy yet.

**Review by use-to — APPROVED** [review]

File: development/CONVENTIONS.md (hunk 1 above)
Comment (use-to) [reply development/CONVENTIONS.md]: Everyone is able to fulfill this requirement while using git squashing. I don't see any reason not implementing this style.

**Pull request merged**